### ISSUE TITLE
`const` in `node.go` is fragile to reordering

### ISSUE THREAD

**Issue by thedavidmeister** [holochain/holochain-proto#713]
Noticed in #698

<img width="465" alt="git__ ___go_src_github_com_holochain_holochain-proto" src="https://user-images.githubusercontent.com/629710/40305184-c6da8212-5d3c-11e8-93d2-e307d7da3d5f.png">

In `node.go` the constants are grouped by purpose, e.g. "DHT messages", "Validate messages", etc.

But the value of the constants is determined by their ordering.

For example, adding `MIGRATE_REQUEST` before `PUT_REQUEST` causes tests to fail due to `PUT_REQUEST` changing from `2` to `3`.

Most of the constants have no tests ensuring their value, so a change of order can accidentally revalue some or all the constants.

These constants should ideally not be changing across versions.

**Comment by zippy** [comment]
Yes, this is true.  They get a sequence value from the `iota`, and they actually are very sensitive because they implicitly part of the line protocol as remote nodes use the values to figure out which network action is meant.  In a way the current tests which use "%v" to print out the values of a packet actually do test this correctly in that they make your code break if you change the ordering.  They just don't do so explicitly so that you know what happened.

**Comment by thedavidmeister** [comment]
there are explicit tests in #698 at least for these consts
